src/main/java/com/streamr/client/options/StreamrClientOptions.java

```diff
@@ -16,6 +17,13 @@
             "?controlLayerVersion=" + ControlMessage.LATEST_VERSION +
             "&messageLayerVersion=" + StreamMessage.LATEST_VERSION;
     private String restApiUrl = "https://www.streamr.com/api/v1";
+
+    private String mainnetRpcUrl = "http://localhost:8545";
+    private String sidechainRpcUrl = "http://localhost:8546";
+    private String sidechainFactoryAddress = "0x4081B7e107E59af8E82756F96C751174590989FE";
```

Comment:
Do we have plans that we'd use these factories for anything other that creating DUs? Maybe it would make sense to rename these to "dataUnionMainnetFactoryAddress"/"dataUnionSidechainFactoryAddress" anyway, if the library user creates only DUs with these factories.

Reply:
yes, but there is no production sidechain at the moment. I don't rely on these values anywhere. I set explicitly in my tests

Reply:
But wouldn't it be more clear that setting name would be `dataUnionMainnetFactoryAddress`, as it is otherwise very confusing what that `mainnetFactoryAddress` setting configures? Also it would make the settings more maintainable as it is very likely that we'll add more factories that serve other purposes in the future. Maybe on next PR?

Reply:
ok, I'll change name